# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/loading/preloader-link-media.tentative.html
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/testing/web-platform/tests/loading/preloader-link-media.tentative.html b/testing/web-platform/tests/loading/preloader-link-media.tentative.html
--- a/testing/web-platform/tests/loading/preloader-link-media.tentative.html
+++ b/testing/web-platform/tests/loading/preloader-link-media.tentative.html
@@ -1,9 +1,10 @@
 <!doctype html>
+<meta charset="utf-8">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="resources/dummy.js?pipe=trickle(d1)"></script>
 <link rel="stylesheet" media="print" href="resources/dummy.css?print">
 <link rel="stylesheet" href="resources/dummy.css?non-print">
 <script>
   let t = async_test("Non-matching link media is not preloaded");
   window.addEventListener("load", t.step_func_done(function() {